verification/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -159,10 +164,15 @@ def get_proof_dirs(proof_root, proof_list, proof_marker):
         sys.exit(1)
 
 
-def run_build(litani, jobs):
+def run_build(litani, jobs, factor):
     cmd = [str(litani), "run-build"]
     if jobs:
         cmd.extend(["-j", str(jobs)])
+    else:
+        cpu_num  = os.cpu_count()
+        if cpu_num is not None and cpu_num > 1:
+            lim_jobs = int(factor * cpu_num)
+            cmd.extend(["-j", str(lim_jobs)])
```

Comment:
I think there should be an `else` case. At least log a warning, so that we can see in the logs if we're running on a platform where `cpu_num` is None.

Also, I still think this is incorrect. If `cpu_num` is 2 and the user passes `factor` as 0.25, then `lim` will still be zero. I think you should do `lim_jobs = lim_jobs if lim_jobs else 1`.